uint/src/uint.rs

```diff
@@ -50,6 +50,26 @@ macro_rules! impl_map_from {
 	}
 }
 
+#[macro_export]
+#[doc(hidden)]
+macro_rules! impl_try_from_for_primitive {
+	($from:ident, $to:ty) => {
+		impl $crate::core_::convert::TryFrom<$from> for $to {
+			type Error = &'static str;
+
+			#[inline]
+			fn try_from(u: $from) -> Result<$to, &'static str> {
```

Comment:
Should we perhaps return [TryFromIntError](https://doc.rust-lang.org/std/num/struct.TryFromIntError.html) instead of a static str?

Reply:
@dvdplm It seems `TryFromIntError` cannot be instantiate here as [it has private fields](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=406c395db75c555856f4aeacf8ed8795) and no other constructor methods.

Reply:
I think you're right. That is unfortunate, sorry for wasting your time!

Ok, I guess we'll keep the static str then.

Reply:
Technically, you [can](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=91755f834e4a0a9d8b5241bb0de546cf) create one, but, well, it's not recommended :)

Reply:
Cool, let's keep the static str then :)

Reply:
@dvdplm I guess this PR is ready to be merged then. Or something else I could improve? :]